src/misc/tokenTags.ts

```diff
@@ -0,0 +1,22 @@
+import { TokenInfo } from '@solana/spl-token-registry';
+
+export const checkIsStrictOrVerified = (tokenInfo: TokenInfo) => {
+  return Boolean(tokenInfo.tags?.includes('verified') || tokenInfo.tags?.includes('strict'));
```

Comment:
Actually, what are the criteria for the token to be `verified` or `strict`? Is this documented somewhere else? Trying to understand more

Reply:
Both `verified`, `strict`, and the upcoming `community`

Reply:
will migrate to `community` once it's ready